### DISCUSSION TITLE
Controls description

### DISCUSSION THREAD

**Discussion by johny-mnemonic** [giejay/Immich-Android-TV#51] [Q&A]
Hi. Is there a list of controls ad how they are mapped?
I was not able to find which keys control zoom and rotation of the image.

**Comment by giejay** [comment]
There are no keys to control zoom or orientation

**Comment by johny-mnemonic** [comment]
Ah, sorry.
I haven't encountered image viewing app that wouldn't have such basic functions, yet.
Are these still on the TODO list so one can track it?

**Comment by giejay** [comment]
No, not on my roadmap at least. Maybe someone else can create a Pull Request. For now, you can use the Immich web or android/iPhone app to do the rotation

**Comment by johny-mnemonic** [comment]
OK. I am using Kodi app on my TV for image viewing, which can rotate/zoom fine, but it is very slow, so I hoped I would be able to replace it with your app.
Maybe one day...
Thanks for creating this app anyway!
